index.html

```diff
@@ -2223,14 +2226,47 @@ <h3>Presentations</h3>
 
         <p>
 The contents of the <code>verifiableCredential</code> <a>property</a> shown
-above are <a>verifiable credentials</a>, as described by this specification. The
-contents of the <code>proof</code> <a>property</a> are proofs, as described by
-the Data Integrity [[VC-DATA-INTEGRITY]] specification. An example of a
-<a>verifiable presentation</a> using the JWT proof mechanism is
+above are <a>verifiable credential</a> graphs, as described by this
+specification. The contents of the <code>proof</code> <a>property</a> are proof
+graphs, as described by the Data Integrity [[VC-DATA-INTEGRITY]] specification.
+An example of a <a>verifiable presentation</a> using the JWT proof mechanism is
 provided in the Securing Verifiable Credentials using JOSE and COSE
 [[?VC-JOSE-COSE]] specification.
         </p>
 
+        <section>
+          <h4>Verifiable Credential Graphs</h4>
+
+          <p>
+When expressing <a>verifiable credentials</a> in a <a>presentation</a>, it is
+important to ensure that data in one <a>verifiable credential</a> is not
+mistaken to be the same data in another <a>verifiable credential</a>. For
+example, if one has two <a>verifiable credentials</a>, each containing an
+object of the following form: `{"type": "Person", "name": "Jane Doe"}`, it is
+not possible to tell if one object is describing the same person as the other
+object. In other words, prematurely merging data between two
+<a>verifiable credentials</a> can lead to a corrupted data model.
+          </p>
+
+          <p>
+In order to ensure that data from different <a>verifiable credentials</a> are
+not accidentally co-mingled, the concept of a
+<dfn class="lint-ignore">verifiable credential graph</dfn> is utilized to
+encapsulate the <a>verifiable credential</a>. The value(s) associated with the
+`verifiableCredential` property on a <a>presentation</a> is of type
+<dfn class="lint-ignore">VerifiableCredentialGraph</dfn>. Using this type has a
+concrete effect on full JSON-LD processing, which properly separates graph
+node identifiers in one graph vs. another graph. Implementers that do not do
+full JSON-LD processing will need to keep this in mind if they merge data
+from one <a>verifiable credential</a> into another, such as when the
+`credentialSubject.id` is the same in both <a>verifiable credentials</a> but
+the object might contain objects of the "Jane Doe" form described in the
+previous paragraph. Care is to be taken to not merge objects that use
+document-relative identifiers, such as blank node identifiers, and only
```

Comment:
Is the term 'blank node identifier' used in the specification? This sentence is meant to non JSON-LD users and, for them, this term is meaningless.

Reply:
This has been fixed in d5437790f66320dc6cb0ad8b24499b6c62f10bf9.